docs/best-practices/accessibility.md

```diff
@@ -0,0 +1,31 @@
+<frontmatter>
+  title: "Best Practices: Accessibility"
+</frontmatter>
+
+# Accessibility Best Practices
+
+TEAMMATES aims to be compliant to the [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/WAI/standards-guidelines/wcag/). On top of the full list of WCAG guidelines, we have compiled some general guidelines below.
+
+## Content
+
+* Language used should be plain and easy to understand. Avoid using complicated figures of speech.
+* Content flow should be logical.
+* Headings should be used in a logical sequence (e.g. descending order, no skipping levels) to split the page into sections, and not used purely for visual design.
+
+## Images/Icons
+
+* Images that convey meaning should have a descriptive `alt` attribute.
+* Decorative images should have an empty `alt` attribute.
+* Icons used in controls (e.g. buttons) should have a descriptive label if they convey meaning, and should be hidden from assistive technologies (e.g. `aria-hidden` attribute set to `true`) if they are purely descriptive.
+
+## Keyboard
+
+* All interactable elements (e.g. buttons, links, inputs) should be focusable by the keyboard (e.g. when tabbing).
```

Comment:
Should **interactable** be reworded to **interactive** instead?

Reply:
👍 Looks like the WCAG page uses interactive elements (over interactable elements) also, would be good to standardise

Reply:
Good catch!